README.md

```diff
@@ -1,13 +1,3 @@
-# retest-model
+This project is inactive and archived (DO NOT DELETE, we might want to access old versions in the repository).
```

Comment:
Can't we just move this to an other/private repo and redirect to recheck, do we need to have this public?

Reply:
I would archive the repo as soon as we have fully integrated retest-model into recheck and have a first release.